src/public/css/styles.css

```diff
@@ -0,0 +1,139 @@
+/* app colors */
+/* font sizes */
+@font-face {
+  font-family: "Noto";
+  src: url(/public/fonts/NotoSans-Regular.ttf) format("truetype");
+  font-weight: normal; }
+
+.theme, body {
+  background-color: #F8F8F8;
+  font-family: Noto, sans-serif; }
+
+.btn, .login-btn {
+  border: none;
+  border-radius: 4px;
+  font-size: 1em;
+  cursor: pointer;
+  text-decoration: none;
+  box-shadow: 0px 4px 10px 0px rgba(0, 0, 0, 0.5); }
```

Comment:
Can we have the login button on the same elevation as the card it's on? Dropping a shadow from the login button makes it look a level higher than the other elements on the card.